docs/source/optimization_ov.mdx

```diff
@@ -112,18 +118,104 @@ metrics = trainer.evaluate()
 trainer.save_model()
 ```
 
+
+### Joint Pruning, Quantization and Distillation (JPQD)
+
+Other than quantization, compression methods like pruning and distillation are common in further improving the task performance and efficiency. Structured pruning slims a model for lower computational demands while distillation leverages knowledge of a teacher, usually, larger model to improve model prediction. Combining these methods with quantization can result in optimized model with significant efficiency improvement while enjoying good task accuracy retention. In `optimum.openvino`, `OVTrainer` provides the capability to jointly prune, quantize and distill a model during training. Following is an example on how to perform the optimization on BERT-base for the sst-2 task.
```

Comment:
I think we should put a disclaimer here about what kind of model architectures are supported for now.

Reply:
Agree, added in this commit https://github.com/huggingface/optimum-intel/pull/150/commits/18dba544fa0a0b42270de282307b86e6b6c010a1

Reply:
for easy reference/readibility, go to https://moon-ci-docs.huggingface.co/docs/optimum-intel/pr_150/en/optimization_ov after next workflow completes